paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/PromoBadgeScreenshotTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import org.junit.Rule
+import org.junit.Test
+import java.util.Locale
+
+internal class PromoBadgeScreenshotTest {
+
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        PaymentSheetAppearance.entries,
```

Comment:
With all the other screenshots, we might not even need this. But I'm fine with leaving it here.